src/Main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './App.css';
+import HornedBeasts from './HornedBeasts'
+import imageTriceratops from './imgs/triceratops.png'
+import imageRhino from './imgs/rhino.png'
+import imageNarwhal from './imgs/narwhal.jpg'
+
+class Main extends React.Component {
+  render() {
+    return (
+      <>
+        <HornedBeasts Title={"Triceratops"} Description={"A dinosaur with three horns on its head."} imageURL={imageTriceratops} className="dinosaur" />
```

Comment:
Just as a point of style, props are generally lower case. So conventionally, "Title" would be "title". Right now you have some upper case and some lower case.